### DISCUSSION TITLE
Should `shouldComponentUpdate` and similar hooks be flagged?

### DISCUSSION THREAD

**Discussion by lilnasy** [krausest/js-framework-benchmark#1155] [General]
I took inferno implementation's performance to mean that their virtual diffing is somehow faster than non-vdom libraries. I realised it's not representative of the library itself after I read the source and saw `onComponentShouldUpdate` statements. Should there be a relevant flag?

**Comment by fabiospampinato** [comment]
I think there are lots of little gray-area things like that throught implementations in this benchmark, I think those are mostly covered by the "code" link. That something is fast in the benchmark by itself doesn't mean a whole lot, if you are evaluating whether Inferno or something else could be your next framework you should click on the "code" link and check out for yourself if that looks like code you might want to write.

**Comment by lilnasy** [comment]
@fabiospampinato That's a good idea and has been my approach so far. Adding links to the implementations was so useful. There are important factors that you'd only see in the code, like clarity and simplicity of the toolchain, at which voby does a fantastic job. Thank you for making it open source.

**Comment by krausest** [comment]
I think that's a perfect first topic for discussions. I'll move it there.
